crates/lakekeeper/src/catalog/tabular.rs

```diff
@@ -36,29 +36,51 @@ macro_rules! list_entities {
                     query,
                 )
                 .await?;
+                let namespace_id =
```

Comment:
In `list_tables` and `list_views`, where this macro is called, we already have `_namespace_id` available. A Database read is expensive and should be avoided where possible. Lets use the information we already have instead of re-fetching it here.